src/fiatconnect/saga.ts

```diff
@@ -893,97 +893,120 @@ export function* handleFetchFiatConnectProviders() {
   }
 }
 
-export function* handleCreateFiatConnectTransfer({
+/**
+ * Initiates a fiatconnect transfer using the fiatconnect SDK to call a transfer/out or transfer/in endpoint
+ * for a provider.
+ **/
+export function* _initiateTransfer({
```

Comment:
nit: same comment with naming; `_initateTransfer` and `_initiateTransaction` are pretty smiliar names